### PR TITLE
Click Animations

### PR DESCRIPTION
**Pull request by MGleonMG**
- added `ClickAnimation.runCustomClickAnimation()`
    - added to 3 of our label-buttons

### CODE REVIEW

File: src/GUI/Animations/ClickAnimation.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+         * zu wechseln und Animationen wieder zuzulassen
+         */
+        Timer revertScalingTimer = new Timer(animationSpeedInMs, new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                label.setIcon(originalImageIcon);
+                isAnimationBlocked = false;
+                ((Timer) e.getSource()).stop();
+
+                // Im zweiten Parameter kann beliebiger Code asugeführt werden
```
Comment (Syhnx) [inline src/GUI/Animations/ClickAnimation.java]: ...ausgeführt

**Review by Syhnx — APPROVED** [review]
tmm

File: src/GUI/Animations/ClickAnimation.java (hunk 1 above)
Comment (MGleonMG) [reply src/GUI/Animations/ClickAnimation.java]: guter catch

**Review by Syhnx — APPROVED** [review]

**Review by JonasOFFF — COMMENTED** [review]
tmm

**Pull request merged**